### ISSUE TITLE
Social Media Icons are reversed

### ISSUE THREAD

**Issue by phin3has** [draftbox-co/gatsby-attila-theme-ghost#3]
In the navigation: 
https://github.com/armada-inc/gatsby-attila-theme-ghost/blob/master/gatsby-theme-ghost-attila/src/components/navbar.jsx#L63

The social media icons are swapped, Facebook links to Twitter, Twitter Links to Facebook:
```
          <ul className="side-nav">
            <li className="nav-twitter">
              <a
                href={`https://facebook.com/${siteSettings.facebook}`}
                title="Twitter"
                target="_blank"
                rel="noreferrer noopener"
              >
                <i className="icon icon-twitter"></i>
                <span>Twitter</span>
              </a>
            </li>
            <li className="nav-facebook">
              <a
                href={`https://twitter.com/${siteSettings.twitter}`}
                title="Facebook"
                target="_blank"
                rel="noreferrer noopener"
              >
                <i className="icon icon-facebook"></i>
                <span>Facebook</span>
              </a>
            </li>
          </ul>
```

**Comment by shyamlohar** [comment]
Hey @awhitehatter  Thank you for reporting issue. Fixed it here 7c173fbd0b5120395eee5894263d4aa5486d74d2😄

**Comment by thandaanda** [comment]
@awhitehatter Thanks for reporting. Fix is now in place.
